overlay/overlay_exe/overlay_exe.rc

```diff
@@ -0,0 +1,44 @@
+#include <winver.h>
+
+#ifndef DEBUG
+#define VER_DEBUG                   0L
+#else
+#define VER_DEBUG                   VS_FF_DEBUG
+#endif
+
+#ifdef SNAPSHOT_BUILD
+#define	VER_RELEASE	VS_FF_SPECIALBUILD|VS_FF_PRERELEASE
+#else
+#define VER_RELEASE 0L
+#endif
+
+VS_VERSION_INFO VERSIONINFO
+	FILEVERSION		1,3,0,0
+	PRODUCTVERSION	1,3,0,0
+	FILEFLAGSMASK	VS_FFI_FILEFLAGSMASK
+	FILEFLAGS		(VER_DEBUG|VER_RELEASE)
+	FILEOS			VOS_NT_WINDOWS32
+	FILETYPE		VFT_APP
+	FILESUBTYPE		0L
+	BEGIN
+		BLOCK "StringFileInfo"
+		BEGIN
+			BLOCK "040904E4"
+			BEGIN
+				VALUE "CompanyName", "Thorvald Natvig"
```

Comment:
What about establishing “Mumble Developers” like in the copyright notices in code?